### PR TITLE
Update GTTA endpoint

### PR DESCRIPTION
**Pull request by JakeSCahill**
#223 The optional parameter is `reference` and takes only one transaction hash

### CODE REVIEW

File: iri/0.1/references/api-reference.md [hunk 1]
```diff
@@ -1170,7 +1170,7 @@ import json
 command = {
   "command": "getTransactionsToApprove",
   "depth": 15,
-  "references": [
+  "reference": [
```
Comment (thibault-martinez) [inline iri/0.1/references/api-reference.md]: This is just a string, not an array

File: iri/0.1/references/api-reference.md [hunk 2]
```diff
@@ -1197,7 +1197,7 @@ var request = require('request');
 var command = {
   "command": "getTransactionsToApprove",
   "depth": 15,
-  "references": [
+  "reference": [
```
Comment (thibault-martinez) [inline iri/0.1/references/api-reference.md]: Ditto

File: iri/0.1/references/api-reference.md [hunk 3]
```diff
@@ -1229,7 +1229,7 @@ curl http://localhost:14265 \
 -d '{
   "command": "getTransactionsToApprove",
   "depth": 15,
-  "references": [
+  "reference": [
```
Comment (thibault-martinez) [inline iri/0.1/references/api-reference.md]: Ditto

File: iri/0.1/references/api-reference.md [hunk 4]
```diff
@@ -1158,7 +1158,7 @@ Get two consistent tip transaction hashes to use as branch/trunk transactions.
 |Parameter|Required or Optional| Description |Type|
 |--|--|--|--|
 | `depth` |Required| Number of bundles to go back to determine the transactions for approval. |integer|
-| `references` |Optional| Transaction hashes from which to start the weighted random walk. Use this parameter to make sure the returned tip transaction hashes approve a given transaction. |array of strings
+| `reference` |Optional| Transaction hash from which to start the weighted random walk. Use this parameter to make sure the returned tip transaction hashes approve a given reference transaction. |array of strings
```
Comment (thibault-martinez) [inline iri/0.1/references/api-reference.md]:
```suggestion
| `reference` |Optional| Transaction hash from which to start the weighted random walk. Use this parameter to make sure the returned tip transaction hashes approve a given reference transaction. |string
```

**Comment by JakeSCahill** [comment]
Updated @thibault-martinez Thanks

**Pull request merged**